Ast/include/Luau/Lexer.h

```diff
@@ -187,6 +187,11 @@ class Lexer
     static bool fixupQuotedString(std::string& data);
     static void fixupMultilineString(std::string& data);
 
+    int getOffset() const
```

Comment:
```suggestion
    unsigned int getOffset() const
```

The type of `offset` is `unsigned int`.

Reply:
Oh right sorry, fixed it!